util/mk-fipsmodule-cnf.pl

```diff
@@ -8,9 +8,14 @@
 
 use Getopt::Long;
 
-my $activate = 1;
+# Module options for pedantic FIPS mode
+# self_test_onload happens if install_mac isn't included, don't add it below
 my $conditional_errors = 1;
 my $security_checks = 1;
+my $ems_check = 1;
+my $drgb_no_trunc_dgst = 1;
```

Comment:
I'm not sure these default values are right as this creates the fipsmodule.cnf to be used by make install.

Reply:
Fixed, along with the test that used the wrong one.